src/Intro/Intro.tsx

```diff
@@ -46,70 +50,62 @@ const styles: any = (theme: any) => ({
   },
 });
 
-function withStylesTyped(myStyles: any) {
-  return (Component: any) => {
-    return withStyles(myStyles)(Component) as any;
-  };
-}
+// FIXME: any type
+const intersperse = (arr: JSX.Element[], sep: string) =>
+  arr
+    .reduce((acc: JSX.Element[], prev: any) => [...acc, prev, sep], [])
```

Comment:
Shouldn’t `prev` be a `JSX.Element` here?

Reply:
The problem seems to be that `sep` is added to the array too, which is of type string (I assumed). I can't find a typing that works, that was indeed one of my questions :)